### PR TITLE
Correct the docker-compose build

### PR DESCRIPTION
**Pull request by napnap75**

### CODE REVIEW

File: docker-compose/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ FROM arm32v6/alpine:latest
 
 COPY --from=builder /tmp/qemu-arm-static /usr/bin
 
-RUN apk add --no-cache py3-pip py3-cffi \
+RUN apk add --no-cache py2-pip py2-cffi \
```
Comment (SuperSandro2000) [inline docker-compose/Dockerfile]: I am interested why you downgraded to python 2?

File: docker-compose/Dockerfile [hunk 2]
```diff
@@ -9,7 +9,7 @@ FROM arm32v6/alpine:latest
 
 COPY --from=builder /tmp/qemu-arm-static /usr/bin
 
-RUN apk add --no-cache py2-pip \
+RUN apk add --no-cache py2-pip py2-cffi py2-cparser \
 	&& pip install docker-compose
```
Comment (SuperSandro2000) [inline docker-compose/Dockerfile]: It is not worth to add ``-U`` is it?

File: docker-compose/Dockerfile (hunk 1 above)
Comment (napnap75) [reply docker-compose/Dockerfile]: Because pip is only available in python 2 in the alpine packages

Comment (SuperSandro2000) [reply docker-compose/Dockerfile]: Strange. I wonder why.

Comment (SuperSandro2000) [reply docker-compose/Dockerfile]: FYI pip3 gets shipped with python3. Just run ``apk add python3`` and you have it.

**Pull request merged**